Utilities/ConsoleTable.cs

```diff
@@ -97,14 +97,14 @@ private static void WriteTableEntry(string key, string value)
             System.Console.WriteLine(footer);
         }
 
-        private static bool WordCanFitOnLine(string Word, int maxWidth, int charactersUsedSoFar)
+        private static bool WordCanFitOnLine(string word, int characterCount)
         {
-            return (Word.Length + charactersUsedSoFar < maxWidth - 2);
+            return (word.Length + characterCount < fixedWidth);
         }
 
-        private static int NumberOfCharsLeftOnLine(int maxWidth, int charactersUsedSoFar)
+        private static int NumberOfCharsLeftOnLine(int characterCount)
         {
-            return (maxWidth - 2) - charactersUsedSoFar;
+            return fixedWidth - characterCount;
```

Comment:
Does the -2 come into the fixedWidth here? just want to make sure all is working as normal

Reply:
Yes. MaxWidth is only used in the definition of rightSpacing. Functionality has remained as it was in #4 with the fix of #5

Reply:
Righeto, as long as you're sure functionality will be as normal with this change, I'll merge now as I have no further questions.